yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/util/AnchorType.java

```diff
@@ -0,0 +1,16 @@
+package com.yoti.api.client.spi.remote.util;
+
+public enum AnchorType {
+
+    SOURCE("SOURCE", "1.3.6.1.4.1.47127.1.1.1"),
+    VERIFIER("VERIFIER", "1.3.6.1.4.1.47127.1.1.2"),
+    UNKNOWN("UNKNOWN", "");
+
+    public final String stringValue;
+    public final String extensionOid;
+
+    AnchorType(String stringValue, String enxtensionOid) {
+        this.stringValue = stringValue;
+        this.extensionOid = enxtensionOid;
```

Comment:
`extensionOid`, is it on purpose or is it a typo?

Reply:
it's not a typo. It stands for object identifier.

Reply:
Ah, ok :+1:

Reply:
why not extOjbIdentifier ?